### DISCUSSION TITLE
I need some advice on json files

### DISCUSSION THREAD

**Discussion by Van4kk** [Laravel-Lang/common#23] [Q&A] [answered]
### **Yo !** 

I'm using  **Laravel Lang** json files an I need an advice or explication how to use ":variable".

I know that I can make a sentence universal by changing a ":word", but where do I write that ":word", how do I use it?

Thank you !

**Comment by andrey-helldar** [comment, marked as answer]
Hi!

For example, you need to add your key to the localization file, for example, `de`:

- Laravel <= 8: `resources/lang/de.json`
- Laravel 9+: `lang/de.json`

```json
{
    "The :word is only part of the sentence.": "Das :word ist nur ein Teil des Satzes."
}
```

And in the application call by key:

```php
__('The :word is only part of the sentence.', ['word' => 'foo'])
// returns "Das foo ist nur ein Teil des Satzes."
```

See https://laravel.com/docs/9.x/localization

**Reply by Van4kk** [reply]
Thank you very much !!! ❤️
